### ISSUE TITLE
data explorer file boxes need to be expandable (Trac #688)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#817]
the data and theory boxes in the data explorer are scrollable but currently not expandable so if one has a large screen and lots of files loaded you never can see more than the small number of files (4 on my machine) at one time.  This is VERY annoying when using heavily -- probably an easy fix too?

Migrated from http://trac.sasview.org/ticket/688
```json
{
    "status": "closed",
    "changetime": "2018-03-04T17:06:26",
    "_ts": "2018-03-04 17:06:26.893828+00:00",
    "description": "the data and theory boxes in the data explorer are scrollable but currently not expandable so if one has a large screen and lots of files loaded you never can see more than the small number of files (4 on my machine) at one time.  This is VERY annoying when using heavily -- probably an easy fix too?",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView GUI Enhancements",
    "time": "2016-10-04T16:59:58",
    "component": "SasView",
    "summary": "data explorer file boxes need to be expandable",
    "priority": "minor",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "piotr",
    "type": "enhancement"
}
```

**Comment by rozyczko** [comment]
Trac update at `2016/10/06 17:54:37`: 
* **piotr** changed owner from "" to "piotr"
* **piotr** changed status from "new" to "assigned"

**Comment by rozyczko** [comment]
Trac update at `2016/10/06 17:57:15`: 
* **piotr** changed resolution from "fixed" to ""
* **piotr** changed status from "closed" to "reopened"

**Comment by rozyczko** [comment]
Trac update at `2016/10/06 17:57:37`: **piotr** commented:

closed wrong ticket. oops.

**Comment by rozyczko** [comment]
Trac update at `2016/11/09 10:26:48`: 
* **piotr** changed resolution from "" to "fixed"
* **piotr** changed status from "reopened" to "closed"

**Comment by pkienzle** [comment]
Trac update at `2017/03/03 22:07:00`: 
* **pkienzle** commented:

Some versions of Windows are displaying visual artifacts with the horizontal splitter, but not all.  Maybe a question of setting the correct wx version.  The current status is good enough for 4.1, so changing the ticket to minor issue to be fixed in a release beyond 4.2.


* **pkienzle** changed milestone from "SasView 4.1.0" to "SasView Next Release +1"
* **pkienzle** changed priority from "major" to "minor"

**Comment by butlerpd** [comment]
Trac update at `2018/03/04 17:06:26`: 
* **butler** commented:

this was not reopened so should have stayed in 4.1.0 milestone


* **butler** changed milestone from "SasView Next Release +1" to "SasView 4.1.0"
